MIGRATING_TO_V8.0.0.MD

```diff
@@ -207,3 +207,13 @@ These are the affected properties, listed by the relevant SDK method:
   - Response: `ALL KEYS`
 - `exh.templates.resolveAsJsonUsingCode`
   - Response: `ALL KEYS`
+
+## Unauthenticated requests
+For all [unauthenticated methods](#unauthenticated-methods), in v8.0.0 the response will now correctly undergo the same transformation steps as any other request.
+
+Noticeably their timestamps previously returned as strings are now converted to date objects, the types of these methods have been updated to reflect this change.
+
+### Unauthenticated Methods
+- `exh.auth.authenticate`
+- `exh.users.createAccount`
+- `exh.auth.confirmMfa` for OAuth1 clients
```

Comment:
Although it's technically more correct how would we feel about removing `for OAuth1 clients`?

It's kinda ugly and well the OAuth2 version doesn't return timestamps anyways... 🤷

Reply:
Also okay for me! Can you move this line to be next to the `authenticate` then though? Sorry for the nitpicking 😅